tests/light/functional_tests/destination_drivers/clickhouse_destination/inputs_and_outputs_for_complex_types.py

```diff
@@ -0,0 +1,375 @@
+    "data_smallint_signed": MAX_INT16,
+    # UInt16:  SMALLINT UNSIGNED
+    "data_uint16": MAX_UINT16,
+    "data_smallint_unsigned": MIN_UINT16,
+    # Int32:   INT, INTEGER, MEDIUMINT, MEDIUMINT SIGNED, INT SIGNED, INTEGER SIGNED
+    "data_int32": MAX_INT32,
+    "data_int": MIN_INT32,
+    "data_integer": MAX_INT32,
+    "data_mediumint": MIN_INT32,
+    "data_mediumint_signed": MAX_INT32,
+    "data_int_signed": MIN_INT32,
+    "data_integer_signed": MAX_INT32,
+    # UInt32:  MEDIUMINT UNSIGNED, INT UNSIGNED, INTEGER UNSIGNED
+    "data_uint32": MAX_UINT32,
+    "data_mediumint_unsigned": MIN_UINT32,
+    "data_int_unsigned": MAX_UINT32,
+    "data_integer_unsigned": MIN_UINT32,
+    # Int64:   BIGINT, SIGNED, BIGINT SIGNED, TIME
+    "data_int64": MAX_INT64,
+    "data_bigint": MIN_INT64,
+    "data_signed": MAX_INT64,
+    "data_bigint_signed": MIN_INT64,
+    # UInt64:  UNSIGNED, BIGINT UNSIGNED, BIT, SET
+    "data_uint64": UINT64,
+    "data_unsigned": MIN_UINT64,
+    "data_bigint_unsigned": UINT64,
+    "data_bit": MIN_UINT64,
+    "data_set": UINT64,
+    # Float32: FLOAT, REAL, SINGLE
+    "data_float32": MAX_FLOAT32,
+    "data_float": MIN_FLOAT32,
+    "data_real": MAX_FLOAT32,
+    "data_single": MIN_FLOAT32,
+    # Float64: DOUBLE, DOUBLE PRECISION
+    "data_float64": MAX_FLOAT64,
+    "data_double": MIN_FLOAT64,
+    "data_double_precision": MAX_FLOAT64,
+    # String: LONGTEXT, MEDIUMTEXT, TINYTEXT, TEXT, LONGBLOB, MEDIUMBLOB, TINYBLOB, BLOB, VARCHAR, CHAR, CHAR LARGE OBJECT, CHAR VARYING, CHARACTER LARGE OBJECT, CHARACTER VARYING, NCHAR LARGE OBJECT, NCHAR VARYING, NATIONAL CHARACTER LARGE OBJECT, NATIONAL CHARACTER VARYING, NATIONAL CHAR VARYING, NATIONAL CHARACTER, NATIONAL CHAR, BINARY LARGE OBJECT, BINARY VARYING
+    "data_binary_large_object": INPUT_STRING,
+    "data_binary_varying": INPUT_STRING,
+    "data_blob": INPUT_STRING,
+    "data_char_large_object": INPUT_STRING,
+    "data_char_varying": INPUT_STRING,
+    "data_char": INPUT_STRING,
+    "data_character_large_object": INPUT_STRING,
+    "data_character_varying": INPUT_STRING,
+    "data_longblob": INPUT_STRING,
+    "data_longtext": INPUT_STRING,
+    "data_mediumblob": INPUT_STRING,
+    "data_mediumtext": INPUT_STRING,
+    "data_national_char_varying": INPUT_STRING,
+    "data_national_char": INPUT_STRING,
+    "data_national_character_large_object": INPUT_STRING,
+    "data_national_character_varying": INPUT_STRING,
+    "data_national_character": INPUT_STRING,
+    "data_nchar_large_object": INPUT_STRING,
+    "data_nchar_varying": INPUT_STRING,
+    "data_string": INPUT_STRING,
+    "data_text": INPUT_STRING,
+    "data_tinyblob": INPUT_STRING,
+    "data_tinytext": INPUT_STRING,
+    "data_varchar": INPUT_STRING,
+    "data_bool": 1,
+    "data_uuid": "123e4567-e89b-12d3-a456-426614174000",
+    "data_ipv4": "192.168.1.1",
+    "data_date": 17897,
+    "data_date32": 47482,
+    "data_datetime": 1546300800,
+    "data_datetime64": 1546300800,
```

Comment:
Setting a `DateTime64` from a string would be really important.
Can we add one more column that does this? Thanks!

Reply:
thanks, added with
```
("data_datetime64_string", "DateTime64(3, 'Europe/Budapest')"), -> create table schema
string data_datetime64_string = 77; -> proto file update part
```

```
"data_datetime64_string": '2019-01-01 00:00:00', -> input
"data_datetime64_string": '2019-01-01 01:00:00.000', -> expected output
```